README.md

```diff
@@ -26,6 +26,7 @@ Options:
   --queue-table TEXT  Queue table name (Default: os.getenv("DBTABLE",
                       "public.modngarn_job"))
   --limit INTEGER     Limit jobs (Default: 300)
+  --max-delay TEXT    Max delay for failed jobs (seconds)(Default: None)
```

Comment:
Should the type not be INTEGER?

Reply:
I think it should be `Decimal`.

Reply:
@KayEss ...is it OK to use `float` in this circumstance? Python's datetime.timedelta(seconds=delay) doesn't support `Decimal` so I have to convert it to float anyway. T T

Reply:
Yup, float is fine. Precision isn't really a factor here.